scripts/build_vcpkg_deps.ps1

```diff
@@ -71,15 +71,16 @@ function run_script {
         run_command -command "git clone $vcpkg_git_url $dest_folder_name"
         Set-Location -Path $dest_folder_path
         run_command "git checkout $VcpkgCommitSha"
-    } else {
+    }
+    else {
```

Comment:
Why did you change the formatting here (and at the catch)?

Reply:
The auto formatting in VS Code is set to do this automatically, I'm going to be cheeky and leave it otherwise I will have to remember to undo this each time I open this file